app/experimenter/experiments/api/v2/serializers.py

```diff
@@ -265,6 +263,11 @@ def update(self, instance, validated_data):
 
             raise serializers.ValidationError({"variants": error})
 
+    def update(self, instance, validated_data):
+        instance = self.update_instance(instance, validated_data)
```

Comment:
You're calling `super.update()` inside of here which is a bit wonky and indirect, can you move the super call into here and just call `update_instance` after?